.env.skel

```diff
@@ -1,4 +1,3 @@
-APP_NAME=""
-APP_PORT=8080
-SERVER_NAME=localhost
-
+APP_NAME="planit"
+SERVER_PORT=8080
+CONFIG_PATH=""
```

Comment:
let's include a dummy config_path. I'm not sure what should I put here.

CONFIG_PATH="config/config.yaml"

Reply:
I removed the yaml file entirely. Instead we are going to use typed-config. We also do not need this config anymore